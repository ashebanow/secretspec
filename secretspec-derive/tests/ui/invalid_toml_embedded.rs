use secretspec_derive::define_secrets;

// This should fail because the TOML is invalid
define_secrets!("invalid_toml_embedded.txt");

fn main() {}