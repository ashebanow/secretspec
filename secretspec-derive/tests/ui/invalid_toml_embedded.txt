this is not valid TOML