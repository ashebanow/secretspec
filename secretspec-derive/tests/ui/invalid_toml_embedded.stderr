error: Failed to parse TOML: I/O error: Failed to resolve path $WORKSPACE/target/tests/trybuild/secretspec-derive/invalid_toml_embedded.txt: No such file or directory (os error 2)
 --> tests/ui/invalid_toml_embedded.rs:4:1
  |
4 | define_secrets!("invalid_toml_embedded.txt");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `define_secrets` (in Nightly builds, run with -Z macro-backtrace for more info)
